letsdrill.rb

```diff
@@ -1,24 +1,42 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
-
+  if integer >= 90
+    'A'
+  elsif integer < 90 && integer >= 80
+    'B'
+  elsif integer < 80 && integer >= 70
+    'C'
+  elsif integer < 70 && integer >= 60
+    'D'
+  elsif integer < 60
+    'F'
+  end
 end
 
 def shortest_string(array)
```

Comment:
there's a few ways in ruby to complete this in a more concise way. try this one again using `.min_by`.
https://ruby-doc.org/core-2.5.0/Enumerable.html#method-i-min_by